### PR TITLE
Management API, not Partner API

### PR DESCRIPTION
**Pull request by cloveras**
Replaced all mentions of the Partner API with the Management API.

Please merge if OK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ since the partner uses
 [partner keys](#partner-keys).
 
 Partner keys can be used with the
-[Partner API](https://developer.vippsmobilepay.com/docs/APIs/partner-api).
+[Management API](https://developer.vippsmobilepay.com/docs/APIs/management-api/.
```
Comment (cloveras) [inline README.md:50]:
```suggestion
[Management API](https://developer.vippsmobilepay.com/docs/APIs/management-api/).
```

File: README.md [hunk 2]
```diff
@@ -259,26 +259,26 @@ See: [How to change partners for a merchant](#how-to-change-partners-for-a-merch
...
 
 See:
-[Submit a product order for a merchant](https://developer.vippsmobilepay.com/docs/APIs/partner-api/vipps-partner-api#submit-a-product-order-for-a-merchant).
+[Pre-fill a product order](https://developer.vippsmobilepay.com/docs/APIs/management-api/management-api-guide/#pre-fill-a-product-order).
 
 **Please note:** A partner that only rarely needs to onboard a new merchant can
 have the merchant order Vipps on
 [portal.vipps.no](https://portal.vipps.no)
-instead of using the Partner API's pre-fill functionality.
+instead of using the Management API's pre-fill functionality.
```
Comment (cloveras) [inline README.md:277]:
```suggestion
instead of using the Management API's "pre-fill" functionality.
```

File: old-signup-api.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ also called "partial signup", is no longer available,
 and partners should use the more efficient
 [partner keys](./partner-keys.md)
 (and the
-[Vipps Partner API](https://developer.vippsmobilepay.com/docs/APIs/partner-api)
+[Management API](https://developer.vippsmobilepay.com/docs/APIs/management-api/
```
Comment (cloveras) [inline old-signup-api.md:18]:
```suggestion
[Management API](https://developer.vippsmobilepay.com/docs/APIs/management-api/)
```

**Review by rebekaburnett — APPROVED** [review]
Great!!

File: README.md [hunk 4]
```diff
@@ -177,10 +177,10 @@ where the merchants can sign up and select you as their partner.
 See:
 [Developer documentation](https://developer.vippsmobilepay.com/docs/vipps-developers).
 
-### Vipps Partner API
+### Vipps Management API
```
Comment (rebekaburnett) [inline README.md:180]:
```suggestion
### Management API
```

**Pull request merged**